src/components/layout/Navigation.js

```diff
@@ -140,7 +140,9 @@ function Navigation(props) {
                                 id={build(ids.NAVIGATION, ids.DATA_BTN)}
                                 disableElevation
                                 className={
-                                    activeView === NavigationConstants.DATA
+                                    activeView.includes(
```

Comment:
It didn't save my comment!

I don't think any of the other navigation buttons are using an `includes` check.  Was there a reason for this change?

Reply:
Yes. Since I introduced dynamic routing, when the page navigate to a path for example `https://localhost:3000/data/ds/iplant/home/sriram`, the activeView is  `/data/iplant/home/sriram`. Earlier it used to just `data` .

Reply:
Eventually, when we introduce dynamic routing for apps, analyses etc...We will have to update the corresponding checks as well.

Reply:
Ah, I didn't realize how that change affected this code. It sounds like we might need more checks though in case someone has a folder/path with `apps` in the name or an app has `data` in the name, etc.

Reply:
Wow, something weird going on with GitHub comments!  I only just saw your second comment which is exactly what I said 😆

Reply:
Ok. I updated to logic to extract active view from the router path and restored existing logic in Navigation. Hopefully this workouts for all the cases.